nixos/modules/tasks/filesystems/btrfs.nix

```diff
@@ -31,5 +68,56 @@ in
       ''
         btrfs device scan
       '';
-  };
+    }
+    // mkIf cfgScrub.enable {
```

Comment:
Does this work as expected, I though you needed to have `mkMerge` or something? In particular, I would assume the `//` will remove the `(any (fs: fs == "btrfs") config.boot.supportedFilesystems)` condition of the above code.

Reply:
This indeed broke my initrd, as I discovered on booting today, argh. 🤦‍ You're totally right.

I updated it to use `mkMerge` in a similar way to what the zfs module does, now I see why they're doing it that way.